### PR TITLE
Fix broken links in flux installation guide

### PR DESCRIPTION
**Pull request by sukrit007**
Fix broken links for controlling flux and more sophisticated setups

### CODE REVIEW

File: site/standalone/installing.md [hunk 1]
```diff
@@ -118,5 +118,5 @@ deployed, give Flux access to it and see modifications land are
 very straight-forward and are a quite natural work-flow.
 
 As a next step, you might want to dive deeper into [how to control
-Flux](site/using.md) or check out [more sophisticated
-setups](site/setup.md).
\ No newline at end of file
+Flux](/site/using.md) or check out [more sophisticated
+setups](/site/setup.md).
```
Comment (dholbach) [inline site/standalone/installing.md]: This should be `/site/standalone/setup.md`.

**Comment by rade** [comment]
btw, would `../using.md` etc work?

**Comment by sukrit007** [comment]
@dholbach  @rade   Changed links to relative links.  `../using.md`  does seem to work

**Comment by dholbach** [comment]
Thanks for the fix! Approved.

**Pull request merged**